### PR TITLE
[SMALLFIX] Object storage test fixes

### PR DESCRIPTION
**Pull request by calvinjia**
Changes some of the default config to make object storage test profiles run more smoothly.

### CODE REVIEW

File: tests/pom.xml [hunk 1]
```diff
@@ -257,13 +257,18 @@
...
-                <!-- Change "s3n://my-bucket/alluxio-test" to the name of your
-                     S3 bucket for this test -->
-                <s3Bucket>s3n://my-bucket/alluxio-test</s3Bucket>
+                <!-- Change "myAccessKey" to your access key for this test if using s3a -->
+                <aws.accessKeyId>myAccessKey</aws.accessKeyId>
+                <!-- Change "mySecretKey" to your secret key for this test if using s3a -->
+                <aws.secretKey>mySecretKey</aws.secretKey>
+                <!-- Change "s3a://my-bucket/alluxio-test" to the name of your S3 bucket & dir -->
+                <s3Bucket>s3a://my-bucket/alluxio-test</s3Bucket>
+                <!-- Increase the buffer time of journal writes to speed up tests -->
```
Comment (madanadit) [inline tests/pom.xml]: Can you please add this to the swiftTest profile as well?. Thanks!

**Comment by ifcharming** [comment]
LGTM

File: tests/pom.xml (hunk 1 above)
Comment (calvinjia) [inline tests/pom.xml]: Updated, thanks. I originally omitted because I thought swift would always be against a simulated cluster.

**Comment by madanadit** [comment]
LGTM

**Pull request merged**